### ISSUE TITLE
integration tests can't be run locally on most systems

### ISSUE THREAD

**Issue by kremalicious** [oceanprotocol/ocean.js#1585]
When following readme and then running `npm run test:integration`:

- on macOS: fails right at first test publishing dataset & algo
- on any VM in GitHub Codespaces: address file is written to inaccessible location, making starting compute job fail in first "Simple compute tests"
  <img width="641" alt="Screen Shot 2022-08-17 at 11 54 22" src="https://user-images.githubusercontent.com/90316/185102819-e02c23c7-9a76-42bb-9fb7-f9532dd585b7.png">

**Comment by alexcos20** [comment]
Did you started barge with --with-c2d and wait until c2d is up? It can take 5 mins  (https://github.com/oceanprotocol/ocean.js/blob/main/.github/workflows/ci.yml#L123-L126)

Pretty sure that you cannot use ARM chipsets, as all of our images are 64bit only

**Comment by kremalicious** [comment]
> Did you started barge with --with-c2d and wait until c2d is up?

yes, doing exactly what our CI does now. Getting different results in different systems and branches. Like, here, nothing really changed the respective tests and they work in `main`, but suddenly do not work anymore in this PR with this error message https://github.com/oceanprotocol/ocean.js/runs/7877933514?check_suite_focus=true#step:12:32

In GitHub codespaces I always can't get the compute envs in that same test, always getting `Failed to establish a new connection: [Errno 111] Connection refused')': /api/v1/operator/environments?chainId=8996`

> Pretty sure that you cannot use ARM chipsets, as all of our images are 64bit only

Not sure if this matters, as Codespaces runs inside a Docker container as far as I understood https://docs.github.com/en/codespaces/overview

But the one I use is x86_64 architecture:
<img width="802" alt="Screen Shot 2022-08-17 at 13 21 14" src="https://user-images.githubusercontent.com/90316/185117327-9c5d983b-4f14-470a-b95d-345883fc4314.png">

**Comment by kremalicious** [comment]
started completely fresh again and I now locally arrive exactly where also the CI fails, wohoo. No clue why, can only assume on my first try something didn't work out with barge, which then also on 2nd run didn't resolve. Only nuking everything including Docker and starting fresh now worked in a GitHub Codespace
